public/scripts/main/services/leftService.js

```diff
@@ -25,24 +23,64 @@ coaxsApp.service('leftService', function (leafletData) {
           color  : '#' + layer.feature.properties.routeColor,
           weight : 1,
         });
-      })
+      });
 
       geoJson.addTo(map);
       cb(geoJson);
-    })
+    });
   };
 
-  // executes the user poi highlight updates
-  this.targetPOIUsers = function (poiUsersCircles, id) {
-    poiUsersCircles.eachLayer(function (layer) {
-      if (layer.options.userId == id) {
-        layer.setStyle({opacity : 1, fillOpacity : 1});
-      } else {
-        layer.setStyle({opacity : 0, fillOpacity : 0});
-      }
-    });
-  }
+   
+  // OLD VERSION (PRE-DAY IN THE LIFE)
+  // this.targetPOIUsers = function (poiUsers, id) {
```

Comment:
please dont comment out huge swaths of code (it is either in the code or it is out - we do not want this on gh)